.yarn/versions/7e2a8b9b.yml

```diff
@@ -0,0 +1,12 @@
+releases:
+  "@radix-ui/react-context-menu": minor
+  "@radix-ui/react-dropdown-menu": minor
+  "@radix-ui/react-menu": minor
+  "@radix-ui/react-radio-group": minor
```

Comment:
Let's mark `react-radio-group` as `minor` and the rest can be `patch`

Reply:
Done.